lib/beaker/cli.rb

```diff
@@ -103,8 +116,6 @@ def execute!
           exit(1)
         end
 
-        provision
```

Comment:
@tvpartytonight I like separating them out. If we have to do a little more work invalidating the `validate` & `configure` parameters for the `execute` subcommand, I think it's worth it. 👍